app/actions/ClientActions.js

```diff
@@ -362,11 +419,23 @@ export const getTransactions = () => async (dispatch, getState) => {
 
 export const NEW_TRANSACTIONS_RECEIVED = "NEW_TRANSACTIONS_RECEIVED";
 
+function checkAccountsToUpdate(txs, accountsToUpdate) {
+  txs.forEach(tx => {
+    tx.tx.getCreditsList().forEach(credit => {if (accountsToUpdate.find(eq(credit.getAccount()))) accountsToUpdate.push(credit.getAccount());});
```

Comment:
You missed a `!` at the `{if (accountsToUpdate.find(...`. Updating the accounts balance for receiving txs is broken without it.

Reply:
ah nice catch